repose-aggregator/artifacts/docker/src/docker/resources/file/ubuntu/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Use Ubuntu as a base, since it works and is familiar.
+# The version should be the latest tested, LTS version.
+FROM ubuntu:16.04
```

Comment:
I wonder when it is time to move to 18.04?

Reply:
I suppose it doesn't matter too much since Repose will run in this environment and Docker isolates the environment to some extent. Even so, I don't believe there is any reason to keep ourselves on 16.04, so we may as well just upgrade for whatever minimal improvements we get from doing so.

I'm tempted to just build the Docker images off of `ubuntu` and `centos` letting the `latest` tag be used. That introduces the  possibility that our build will break, but it's fairly unlikely considering the content of our Dockerfiles.

Well, I'll go ahead and bump to Ubuntu 18.04 for this PR.